x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlCapabilities.java

```diff
@@ -170,7 +170,12 @@ public enum Cap {
          * Fix for non-unique attribute names in ROW and logical plans.
          * https://github.com/elastic/elasticsearch/issues/110541
          */
-        UNIQUE_NAMES;
+        UNIQUE_NAMES,
+
+        /**
+         * MATCH command support
+         */
+        MATCH_COMMAND;
```

Comment:
:+1:

Reply:
Oh! Could you make this `SNAPSHOT only` by adding `(true)`

Reply:
Thanks for the tip! cda519de83be8d1494ba83077779e22ab68e7405